.taskcluster.yml

```diff
@@ -10,13 +10,15 @@ tasks:
     # -- This is where you add new projects -----------------------------------
     # -------------------------------------------------------------------------
     PROJECTS:
-      # [ <PROJECT NAME>, <PYTHON VERSION>, <DOCKERHUB REPO>]
-      - ['client',        '36',             '']
-      - ['client',        '37',             '']
-      - ['iscript',       '36',             '']
-      - ['iscript',       '37',             '']
-      - ['treescript',    '36',             '']
-      - ['treescript',    '37',             ''] # mozilla/releng-treescript'] # until we figure out how to install it
+      # [ <PROJECT NAME>,   <PYTHON VERSION>, <DOCKERHUB REPO>]
+      - ['client',          '36',             '']
+      - ['client',          '37',             '']
+      - ['iscript',         '36',             '']
+      - ['iscript',         '37',             '']
+      - ['treescript',      '36',             '']
+      - ['treescript',      '37',             ''] # mozilla/releng-treescript'] # until we figure out how to install it
+      - ['beetmoverscript', '36',             '']
+      - ['beetmoverscript', '37',             mozilla/releng-beetmoverscript']
```

Comment:
It looks like we need to rename this. `mozilla/releng_beetmoverscript`?

`Invalid repository name (releng-beetmoverscript'), only [a-z0-9-_.] are allowed`

Reply:
ah, that was a typo. i missed one initial `'`